packages/node_modules/@ciscospark/widget-recents/src/components/NoSpaces.css

```diff
@@ -0,0 +1,30 @@
+.noSpacesWrapper {
+  height: 100%;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  background-color: black;
+}
+
+.noSpacesImg {
+  height: 72px;
+  width: 100%;
+  margin-bottom: 24px;
+}
+
+.noSpacesTitle {
+  color: #fff;
+  font-family: CiscoSansTT Bold,Helvetica Neue,Arial;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  margin-bottom: 12px;
+}
+
+.noSpacesMessage {
+  color: hsla(0,0%,100%,.6);
```

Comment:
Is this color code from the web client? Could we use a collab style instead?

Reply:
color code is from web client, not sure what the matching collab style is (if there is one)

Reply:
update: no matching collab-ui style